### ISSUE TITLE
Legacy Mode Fights Do Not Load w/ Custom Fighter

### ISSUE THREAD

**Issue by dimblebutts** [RPCS3/rpcs3#12610]
### Quick summary

If you try and play legacy mode, you can create your fighter and start the mode, but as soon as you load into your first fight the game hangs on the loading screen. Audio continues as does the loading animation in the bottom corner, but the fight never actually loads no matter how long you wait.

### Details

Everything works normally as it does on the PS3 up until you accept and try and fight your first fight in legacy mode. At this point the game hangs indefinitely on the loading screen. Audio does continue without interruption and the loading animation continues to move. Unfortunately the fight will never load. This happens when you try and use a user created fighter in legacy mode. If you use one of the already existing characters in legacy mode, sometimes it will briefly work before eventually crashing/freezing/hanging.

### Attach a log file

[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/9486526/RPCS3.log.gz)

### System configuration

Ryzen 5 3600 16gb ram RTX 3800 Ti
Steam Deck
Intel 8700k 16gb ram RTX 3080 TI
Same results on all 3 systems across multiple drivers

**Comment by digant73** [comment]
@dimblebutts is that still an issue?

**Comment by TheFlash3113** [comment]
@digant73 Yes, still an issue. Same issue with existing fighters. Legacy mode just doesn't work.
